libr/asm/asm.c

```diff
@@ -139,11 +139,7 @@ R_API RAsm *r_asm_new() {
 		return NULL;
 	}
 	for (i=0; asm_static_plugins[i]; i++) {
-		static_plugin = R_NEW (RAsmPlugin);
-		if (!static_plugin) continue;
-		// memleak here
-		memcpy (static_plugin, asm_static_plugins[i], sizeof (RAsmPlugin));
-		r_asm_add (a, static_plugin);
+		r_asm_add (a, asm_static_plugins[i]);
```

Comment:
Same as above, most of these plugins have their `fini` as `NULL`, and the one's which have `fini` doesn't use the passed pointer.